tf/train.py

```diff
@@ -130,11 +136,12 @@ def main(cmd):
         tfprocess.restore(cp)
 
     # Sweeps through all test chunks statistically
-	# Assumes average of 10 samples per test game.
+    # Assumes average of 10 samples per test game.
+    # Evaluation result doesn't depend on batch size, so we can use split instead of total batch size.
```

Comment:
This comment seems potentially confusing.  Maybe focus on the fact that test uses the same split batch size as train for simplicity, then mention that that is okay because test eval result is just an average which works out the same regardless of batch size.